parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -136,18 +136,18 @@ installStandaloneContainerd() {
     CONTAINERD_PATCH_VERSION="${2:-1}"
 
     # runc needs to be installed first or else existing vhd version causes conflict with containerd.
-    ensureRunc ${RUNC_VERSION:-""} # RUNC_VERSION is an optional override supplied via NodeBootstrappingConfig api
+    logs_to_events "AKS.CSE.installContainerRuntime.ensureRunc" ensureRunc ${RUNC_VERSION:-""} # RUNC_VERSION is an optional override supplied via NodeBootstrappingConfig api
```

Comment:
I think this is messed up somehow but can't quite tell how

```
/home/packer/provision_installs_distro.sh: line 246: $1: unbound variable
```

Reply:
I feel like `${@}` must be messing up positional param counting but I don't see it...

Reply:
and bash docs seem to indicate that's what we want, because it interpolates as `"$1" "$2" "$3"` where the alternatives should be `"$1\IFS$2\IFS$3"`

Reply:
yeah i've been looking to repro and sort this out but not there yet. I'm in favor of dropping this log event and knowing that the diff in installStandaloneContainerd vs installStandaloneContainerd.* events is due to ensureRunc. 

I played with the logs_to_events a bit in the ensureRunc func but I question how well the apt_get_install func will work while being wrapped in logs_to_events.

Reply:
ok changed how the ensureRunc func handles $1 and things look happier all around